cmd/operator/main.go

```diff
@@ -144,7 +144,7 @@ var (
 )
 
 func init() {
-	klog.InitFlags(flagset)
+	//klogv2.InitFlags(flagset)
```

Comment:
klog does not need to be initialized (is it done implicitly in gokit)?

Reply:
Independent I think instead of commenting out the initialization it would be great to just leave a human readable comment why initialization can be omitted.